numpy/core/src/common/simd/avx2/operators.h

```diff
@@ -114,6 +114,11 @@ NPY_FINLINE __m256i npyv_shr_s64(__m256i a, int c)
 #define npyv_not_b32 npyv_not_u8
 #define npyv_not_b64 npyv_not_u8
 
+// ANDC, ORC and XNOR
+#define npyv_andc_b8(A, B) _mm256_andnot_si256(A, B)
+#define npyv_orc_b8(A, B) npyv_or_b8(npyv_not_b8(A), B)
+#define npyv_xnor_b8(A, B) npyv_not_b8(npyv_xor_b8(A, B))
```

Comment:
`_cmplt_epi8` and `_cmpeq_epi8` can be used for AVX2 and SSE since boolean values are guaranteed to be 0 or 0xFF, also because {8,16-32}-bit comparison take almost 0.5 latency.

Reply:
Done! I applied the suggested changes above for `xnor`, but not for `orc` since it would require a `less equal` intrinsic (which is not available neither for SSE nor AVX2).